integration_tests/mockApis/curiousApi.ts

```diff
@@ -0,0 +1,107 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+const stubNeurodivergenceForPrisonerWithAllCategoriesOfSupportNeed = (prisonNumber: string): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: `/learnerNeurodivergence/${prisonNumber}`,
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: [
+        {
+          prn: prisonNumber,
+          establishmentId: 'MDI',
+          establishmentName: 'MOORLAND (HMP & YOI)',
+          neurodivergenceSelfDeclared: ['Dyslexia'],
+          selfDeclaredDate: '2022-02-01',
+          neurodivergenceAssessed: ['Attention Deficit Hyperactivity Disorder', 'Alzheimers'],
+          assessmentDate: '2022-02-10',
+          neurodivergenceSupport: ['Communications', 'Visual Support'],
+          supportDate: '2022-02-16',
+        },
+      ],
+    },
+  })
+
+const stubNeurodivergenceForPrisonerWithSelfDeclaredSupportNeeds = (prisonNumber: string): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: `/learnerNeurodivergence/${prisonNumber}`,
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: [
+        {
+          prn: prisonNumber,
+          establishmentId: 'DNI',
+          establishmentName: 'DONCASTER (HMP)',
+          neurodivergenceSelfDeclared: ['ADHD'],
+          selfDeclaredDate: '2022-05-16',
+          neurodivergenceAssessed: ['No Identified Neurodiversity Need'],
+          assessmentDate: '2022-05-16',
+          neurodivergenceSupport: ['No Identified Support Required'],
+          supportDate: '2022-05-16',
+        },
+      ],
+    },
+  })
+
+const stubNeurodivergenceForPrisonerWithNoCurrentAssessment = (prisonNumber: string): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: `/learnerNeurodivergence/${prisonNumber}`,
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: [],
+    },
+  })
+
+const stubNeurodivergenceForPrisonerNotInCurious = (prisonNumber: string): SuperAgentRequest =>
```

Comment:
I think we'll need to handle a 404 and no assessment data in the same way - hence having both stubs